libevm/ExtVMFace.h

```diff
@@ -228,30 +227,32 @@ class EnvInfo
 {
 public:
 	EnvInfo() {}
-	EnvInfo(BlockHeader const& _current, LastHashes const& _lh = LastHashes(), u256 const& _gasUsed = u256()):
+	EnvInfo(BlockHeader const& _current, std::shared_ptr<LastBlockHashesFace const> _lh, u256 const& _gasUsed):
```

Comment:
This code to be exact: https://github.com/ethereum/cpp-ethereum/pull/4126/files#diff-e01cd53eebd917cc35354ffc7719d8adR295

Probably there's a way to change this test code somehow... modifying ´EnvInfo´ is weird in general